index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    
+    <header class="portfolio-header">
+
+    <div class="logo">My Logo</div>
+        <img src="images/Enabled.png" alt="hamburger">
+</header>
+  
+    
+    <div class="container">
+<h1 class="salutation">
+    I’m Alexandra.
+Glad to see you!</h1>
+<p class="developer">
+    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h2 class="connect">LET'S CONNECT</h2>
+
+<div class="social">
+    <ul >
+        <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
+        <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
+        <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
+        <li><a href="https://github.com/ausiemass/portfolio"><img src="images/Vector (1).jpg" alt="GITHUB"></a></li>
+        <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
+    </ul>
+    
+</div>
+
+   
+</div>
```

Comment:
- [ ] [Optional] 👍 Great work on using the `<header>` tag for the header section! However, there are a couple of improvements you can make in the structure of your HTML:

[Optional] 1️⃣ Headline Section: Instead of using a `<div>` tag for your headline section, it is recommended to use a `<section>` tag. The `<section>` tag represents a standalone section of content within a document. It provides better semantic meaning and clarity to your HTML structure.

[Optional] 2️⃣ Main Content: Wrap all the sections, except for the `<header>` and `<footer>`, with a `<main>` tag. The `<main>` tag indicates the main content of the document and helps in structuring your page.

Here is a Good webpage structure
```
<header>
  <h1>Machine Learning Workshop</h1>
  <nav>
  </nav>
</header>
<main>
  <section id="heading">
    <h2>This is heading section</h2>
  </section>

  <section id="about">
    <h2>This is about section</h2>
  </section>

  <section id="feedback">
    <h2>This is feedback section</h2>
  </section>

</main>

<footer>
  <h2>This is a footer</h2>
</footer>

```

By making these updates, you will improve the semantics and structure of your HTML code, making it more understandable and maintainable. Keep up the good work! 👍🌟